examples/oauth/README.md

```diff
@@ -0,0 +1,52 @@
+# Example: `oauth`
+
+This example shows a complete Oauth flow with google. It contains a NextJS app with:
+
+- a frontend application
+- a backend application
+
+This example contains an example auth page as well as a stub API endpoint for "your business" (where the contact is resolved into an organization ID). The creation of the hidden iframe is abstracted by our `@turnkey/iframe-stamper` package. For more information on oauth, [check out our documentation](https://docs.turnkey.com/features/oauth).
+
+## Getting started
+
+### 1/ Cloning the example
+
+Make sure you have `node` installed locally; we recommend using Node v18+.
+
+```bash
+$ git clone https://github.com/tkhq/sdk
+$ cd sdk/
+$ corepack enable  # Install `pnpm`
+$ pnpm install -r  # Install dependencies
+$ pnpm run build-all  # Compile source code
+$ cd examples/oauth/
+```
+
+### 2/ Setting up Turnkey
+
+The first step is to set up your Turnkey organization and account. By following the [Quickstart](https://docs.turnkey.com/getting-started/quickstart) guide, you should have:
+
+- A public/private API key pair for Turnkey
+- An organization ID
+
+Once you've gathered these values, add them to a new `.env.local` file. Notice that your API private key should be securely managed and **_never_** be committed to git.
+
+```bash
+$ cp .env.local.example .env.local
+```
+
+Now open `.env.local` and add the missing environment variables:
+
+- `API_PUBLIC_KEY`
```

Comment:
minor nit: can you update these to reflect what's included in `.env.local.example`?